.github/stale.yml

```diff
@@ -1,28 +1,52 @@
 # Configuration for probot-stale - https://github.com/probot/stale
 
 # Number of days of inactivity before an Issue or Pull Request becomes stale
-daysUntilStale: 60
-# Number of days of inactivity before a stale Issue or Pull Request is closed
-# False means never close
-daysUntilClose: false
+daysUntilStale: 30
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels:
+  - "needs: author reply"
+  - "needs: more work"
+
 # Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
-exemptLabels:
-  - "status: needs review"
-  - "status: needs testing"
-  - "status: needs decision"
-  - "status: ready to merge"
+exemptLabels: []
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: false
```

Comment:
Projects and milestones might be set to true since those are long term issues. Unless we tag them with `kind: todo`?

Reply:
Possible, but probably redundant. Could it happen that we mark for a milestone something that wasn't properly triaged?

Reply:
Dunno. Depends on triaging. We haven't really started doing it. Once the bot is ready, let's modify the PR templates and see how it goes. If needed, we'll just modify it again